File: app/src/main/java/com/katkov/lolachievements/application/ui/checkentryinfo/CheckEntryInfoView.java
package com.katkov.lolachievements.application.ui.checkentryinfo;

import com.arellomobile.mvp.MvpView;
import com.arellomobile.mvp.viewstate.strategy.AddToEndSingleStrategy;
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType;
import com.katkov.lolachievements.domain.model.EntryInfoModel;

public interface CheckEntryInfoView extends MvpView {

    @StateStrategyType(AddToEndSingleStrategy.class)
    void fillInfo(EntryInfoModel entryInfoModel);
}

File: app/src/main/java/com/katkov/lolachievements/application/ui/checkentryinfo/CheckEntryInfoView.kt
package com.katkov.lolachievements.application.ui.checkentryinfo

import com.arellomobile.mvp.MvpView
import com.arellomobile.mvp.viewstate.strategy.AddToEndSingleStrategy
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType
import com.katkov.lolachievements.application.base.ErrorView
import com.katkov.lolachievements.application.base.ProgressView
import com.katkov.lolachievements.domain.model.LoginModel

interface CheckEntryInfoView : MvpView, ErrorView, ProgressView {

    @StateStrategyType(AddToEndSingleStrategy::class)
    fun fillInfo(loginModel: LoginModel?)
}
